### ISSUE TITLE
Fatal error when saving project under Ubuntu Jammy (22.04)

### ISSUE THREAD

**Issue by NetForces** [ImpulseAdventure/GUIslice-Builder#224]
Just a blank project. Right after opening I try to "Save As" and get:

![image](https://user-images.githubusercontent.com/984831/204040063-8d03897a-81a4-4322-b35f-adbdb5107eb4.png)

Log:

```
courchea@X1-Andre:{13:34}~/Downloads/GUIsliceBuilder/logs ➭ cat 20221125_133307.txt
class java.lang.NullPointerException: Cannot invoke "java.io.File.getParent()" because "file" is null
builder@0.17.b16/builder.common.Utils.backupFile(Utils.java:283)
builder@0.17.b16/builder.controller.Controller.saveProject(Controller.java:894)
builder@0.17.b16/builder.controller.Controller.saveAsProject(Controller.java:948)
builder@0.17.b16/builder.views.RibbonListener.actionPerformed(RibbonListener.java:651)
java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2313)
java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:374)
java.desktop/javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:1028)
java.desktop/javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:1072)
java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:298)
java.desktop/java.awt.Component.processMouseEvent(Component.java:6626)
java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3389)
java.desktop/java.awt.Component.processEvent(Component.java:6391)
java.desktop/java.awt.Container.processEvent(Container.java:2266)
java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5001)
java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
java.desktop/java.awt.Component.dispatchEvent(Component.java:4833)
java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2780)
java.desktop/java.awt.Component.dispatchEvent(Component.java:4833)
java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
java.desktop/java.awt.EventQueue$5.run(EventQueue.java:746)
java.desktop/java.awt.EventQueue$5.run(EventQueue.java:744)
java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:743)
java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by Pconti31** [comment]
@NetForces Do a Save before trying SaveAs
I'll check into why saveas of new project fails
Paul--

**Comment by NetForces** [comment]
@Pconti31 yeah that works. Now I can start playing with it ;-)

**Comment by Pconti31** [comment]
Fixed in release 0.17.b17
